### ISSUE TITLE
Can the model predict hand boundary boxes on its own? 

### ISSUE THREAD

**Issue by mpuu00001** [HowieMa/NSRMhand#21]
Hello and thank you for making the code available. I greatly appreciate your efforts.

According to your paper on this work, the NSRMhand model only focuses on hand pose estimation. May I inquire as to whether the proposed methodology is capable of both hand detection and hand landmark localization? Alternatively, does the model need the hand boundary box coordinates or can it predict the boundary boxes on its own? Furthermore, if the model always assumes there is a hand in the input image?

Thank you in advance!

**Comment by HowieMa** [comment]
Yes, as this project aims at pose estimation, we always assume that there is a hand in the input image. For hand detection, you may use OpenPose or some other tools to obtain it.

**Comment by mpuu00001** [comment]
Thanks for your response! 
Is the model only capable of predicting the location of a hand pose (where is the hand) when the input image contains a hand?
